assignments/arrays.js

```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(`Car 33 is a ${inventory[32].car_make} ${inventory[32].car_model} ${inventory[32].car_year}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length-1];
+console.log(lastCar.car_make,lastCar.car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
 console.log();
 
+let sortInv = inventory.sort( (obj1,obj2) => (obj1.car_model > obj2.car_model) ? 1 : -1 );
```

Comment:
Awesome work using the sort method! It looks like you're also familiar with ternary syntax, which is awesome. Another option is to push just the models into the carModels array (which you did) and to just call sort() on the carModels array. Excellent work!